.circleci/config.yml

```diff
@@ -121,15 +134,62 @@ workflows:
   build_and_deploy:
     jobs:
       - build:
+          filters:
+            branches:
+              ignore:
+                - master
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+      - unit_test:
+          requires:
+            - build
+      - deploy_canary:
+          filters:
+            branches:
+              only: develop
+          requires:
+            - unit_test
+      - deploy_branch:
+          filters:
+            branches:
+              ignore:
+                - master
+                - develop
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+          requires:
+            - unit_test
+  build_and_deploy_i18n:
+    jobs:
+      - build_i18n:
           filters:
             tags:
               only: /^v.*/
+            branches:
+              only:
+                - master
+                - /^support.*/
```

Comment:
nit but could we make it match `support\/.*` instead of just `support.*`? Just to not match a
branch like `supportive-boy` and only match things like `support/blah-blah`. same thing with release

Reply:
sure!